### PR TITLE
Move Developer Tools to Settings

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

This moves the Developer Tools from the Left Menu Pinned to the Settings Dashboard

@matthiasdebaat What color should I use?

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -987,6 +987,10 @@
           "settings": {
             "title": "Settings",
             "description": "Basic settings, server controls, logs and info"
+          },
+          "developer_tools": {
+            "title": "Developer Tools",
+            "description": "Try things out quickly"
```
Comment (balloob) [inline src/translations/en.json:993]: People are often scared of the word "developer", so let's describe it better here
```suggestion
            "description": "Tools to help create automations and scripts"
```

**Comment by bramkragten** [comment]
This needs to be updated in the backend too: <https://github.com/home-assistant/core/blob/dev/homeassistant/components/frontend/__init__.py#L375-L383>

**Comment by matthiasdebaat** [comment]
When moving Energy to Dashboards and splitting making Add-ons it's own top level menu item I would like to suggest the following:

- Add-ons `#F1C447` (previously Energy)
- Developer tools `#4084CD` (previously Add-ons, Backups & Supervisor)

**Comment by balloob** [comment]
One thing that is very important is that currently the developer tools DOM element is cached. This means that any state of the user is stored while the frontend is open. We should make sure that feature stays.

**Comment by balloob** [comment]
Since you didn't add code for it, it's not going to be cached. Config needs to be cached and the developer tools page too.

The old code for caching is here https://github.com/home-assistant/frontend/blob/dev/src/layouts/partial-panel-resolver.ts#L21 (And needs to be converted to be config)

**Comment by bramkragten** [comment]
> Since you didn't add code for it, it's not going to be cached. Config needs to be cached and the developer tools page too.
> 
> The old code for caching is here [`dev`/src/layouts/partial-panel-resolver.ts#L21](https://github.com/home-assistant/frontend/blob/dev/src/layouts/partial-panel-resolver.ts?rgh-link-date=2022-01-24T17%3A10%3A01Z#L21) (And needs to be converted to be config)

Actually, since the url did not change, it still works. It is a still a panel, and not part of the config panel. We just removed it from the sidebar.

**Pull request closed without merging**